nemo/collections/common/losses/cross_entropy.py

```diff
@@ -53,6 +53,8 @@ def __init__(self, logits_ndim=2, weight=None, reduction='mean'):
             weight (list): list of rescaling weight given to each class
             reduction (str): type of the reduction over the batch
         """
+        if weight is not None and not torch.is_tensor(weight):
+            weight = torch.FloatTensor(weight).to(self._device)
```

Comment:
we don't need to do `to.device` in PL

Reply:
The weight input should be a Tensor for Pytorch Crossentropy. Does it work if we pass numpy array or regular array? Does it convert it automatically?

Reply:
Will we ever pass numpy or reg arrays?

Reply:
Yes, our code which call this loss is passing numpy array of weights calculated in the data descriptor.